example/lib/main.dart

```diff
@@ -41,6 +41,7 @@ class _MyAppState extends State<MyApp> {
         ),
         body: Column(
           children: [
+            const TextField(), // only used to unfocus the other text field
```

Comment:
I assume this was used for testing? I can't see any benefit of leaving it in the example for explanation purposes. 
Can we remove it if there is no good reason which I'm missing right now?

Reply:
Yes I added this for testing. Because a second `TextField` made it easy to simulate that the normal `TextField` loses the focus. This is great to test what's happening if you click on an the emoji or the backspace button while the `TextField` does not have focus.

But yes, it's ugly. so I will remove it :)